artifactory/commands/transferconfig/transferconfig_test.go

```diff
@@ -99,15 +99,20 @@ func TestSanityVerifications(t *testing.T) {
 	testServer, serverDetails, serviceManager := createMockServer(t, func(w http.ResponseWriter, r *http.Request) {
 		if r.RequestURI == "/api/plugins/execute/checkPermissions" {
 			w.WriteHeader(http.StatusOK)
+		} else if r.RequestURI == "/api/plugins/execute/configImportVersion" {
```

Comment:
Do we have testing code that calls the above `configImportVersion` mock?

Reply:
Yes. The function validateArtifactoryServers looks for the plugin's version and prints it. It must return 200.